src/worker/controller/actions/GetStatusAction.js

```diff
@@ -0,0 +1,58 @@
+const constants = require("../../../common/constants");
+
+class GetStatusAction {
+  constructor(controller) {
+    this._controller = controller;
+  }
+
+  /**
+   * @param {Function} callback (err)=>{}
+   * */
+  async execute(params) {
+    const callback = params.callback;
+    const C = constants.NODE_NOTIFICATIONS;
+    let status = null;
+    let error = null;
+
+    if (this._controller.isWorkerInitInProgress()) {
+      status = constants.WORKER_STATUS.INITIALIZING;
+    } else {
+      let workerParams;
+
+      try {
+        workerParams = await this._controller.asyncExecCmd(C.GET_ETH_WORKER_PARAM);
```

Comment:
`workerParams` can be a `const` defined inside of the `try` clause (it's never used elsewhere).